datadog/fwprovider/data_source_datadog_application_key.go

```diff
@@ -80,7 +81,9 @@ func (d *applicationKeyDataSource) Read(ctx context.Context, req datasource.Read
 			return
 		}
 		apiKeyData := ddResp.GetData()
```

Comment:
```suggestion
		appKeyData := ddResp.GetData()
```

Reply:
Haha, looks like this has been there a long time. I can fix it.